### ISSUE TITLE
when sway `output * scale 2`, waybar renders bigger cursor size than sway and other apps

### ISSUE THREAD

**Issue by iambumblehead** [Alexays/Waybar#1948]
Hello,

I'm setting up waybar for the first time. The system uses sway wm. Before any changes are made, mouse size remains consistently the same over waybar and other apps (qutebrowser and foot terminal emulator). After setting "output * scale 2" inside the sway configuration, cursor size becomes bigger when the mouse hovers over waybar. Is there a way to resolve this? Is there a way to tweak waybar's cursor scaling behaviour specifically, without affecting any other applications?

Thank you

**Comment by iambumblehead** [comment]
glib:bin was installed, to supply the gsettings command to possibly set the cursor size, but this command fails,
```console
gsettings set org.gnome.desktop.interface cursor-size 40
No schemas installed
```

**Comment by iambumblehead** [comment]
I succeeded by doing the following, installing both glib and gsettings schemas, calling gsettings, then uninstalling
```console
guix install glib:bin gsettings-desktop-schemas
gsettings set org.gnome.desktop.interface cursor-size 28
guix remove glib:bin gsettings-desktop-schemas
```

program-generated-systems through guix or nixos won't be able to practically automate this step

**Comment by iambumblehead** [comment]
@BoomerangNebula to match the bigger size of the waybar cursor, it is necessary to set xcursor_theme Adwaita 30 and the cursor is quite huge

Thanks for the reply and suggestion

**Comment by iambumblehead** [comment]
yes as mentioned in the earlier ticket, waybar's cursor can be scaled down with gsettings commands like this one
```
gsettings set org.gnome.desktop.interface cursor-size 28
```

if there could be a way to control waybar's cursor size without gsettings, such as a configuration value, a command modifier, or an env var, that would be great for anyone who does need a different cursor size. It is appreciated that sway allows cursor size to be configured without installing gsettings.

**Comment by mrivnak** [comment]
I'm a bit late to the party here, but worth mentioning as I've spend the past few days trying to solve this on my computer. In order for GTK settings to be applied on wayland you need to install `xdg-desktop-portal-gtk`

**Comment by iambumblehead** [comment]
One can also use a different status bar, such as [yambar,](https://codeberg.org/dnkl/yambar/) which smaller and does not depend on gtk things

**Comment by fxzxmicah** [comment]
> I'm a bit late to the party here, but worth mentioning as I've spend the past few days trying to solve this on my computer. In order for GTK settings to be applied on wayland you need to install `xdg-desktop-portal-gtk`

Thank you, this resolves everything.

**Comment by Alexays** [comment]
Closing — this was resolved/answered in the thread (config, support, or environment, not a Waybar bug still open). Feel free to reopen if you're still stuck. 🙏
